gammapy/irf/tests/test_rad_max.py

```diff
@@ -0,0 +1,33 @@
+import numpy as np
+import astropy.units as u
+from gammapy.maps import MapAxis
+
+
+def test_rad_max_roundtrip(tmp_path):
+    from gammapy.irf import RadMax2D
+
+    n_energy = 10
+    energy_edges = np.geomspace(100 * u.GeV, 100 * u.TeV, n_energy + 1)
+    energy_axis = MapAxis.from_edges(energy_edges, name='energy')
```

Comment:
Here you should rather use `MapAxis.from_energy_bounds` since by default `from_edges` will create a `lin` interpolation (and centers won't be properly calculated).
`energy_axis = MapAxis.from_energy_bounds(100*u.GeV, 100*u.TeV, n_energy, unit="TeV")`

Reply:
done